src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -104,9 +108,55 @@ def __call__(self, organization, event):
                     pass
 
 
+class PullEventWebhook(Webhook):
+    def __call__(self, organization, event):
+        repo = self.repo_from_event(organization, event)
+        pull_request = event['pullrequest']
+
+        integration = Integration.objects.get(
+            id=repo.integration_id,
+        )
+        installation = integration.get_installation(organization_id=organization.id)
+        emails = installation.get_client().get_user_emails()['values']
+        try:
+            commit_author = CommitAuthor.objects.filter(
+                organization_id=organization.id,
+                email__in=[email['email'] for email in emails if email['is_confirmed']],
+            )[0]
+        except IndexError:
+            primary_email = None
+            for email in emails:
+                if email['is_primary']:
+                    primary_email = email['email']
+                    break
+
+            commit_author = CommitAuthor.objects.create(
+                organization_id=organization.id,
+                email=primary_email,
+                name=pull_request['author']['display_name'],
+                external_id=pull_request['author']['uuid'],
+            )
+
+        try:
+            PullRequest.objects.create_or_update(
+                repository_id=repo.id,
+                key=pull_request['id'],
+                values={
+                    'organization_id': organization.id,
```

Comment:
mm having this in `values` rather than above seems wrong ... like in what situation would we want to change the org id...

Reply:
@macqueen This is just copying from github's integration's webhook. see https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github/webhook.py#L322
I'm totally fine with changing it. I'm not entirely familiar with values, but will look into it.

Reply:
@macqueen actually does `update_or_create` exist in django 1.6? Now that I think about it, I thought that was an addition to 1.7....? But still trying to figure out what values means. T.T cc @matt

Reply:
Lemme show you something.

https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/manager.py#L289-L290

Reply:
Oh... no way! lol! XD I should have known haha.

Reply:
yeah what matt said :) 

i feel like it's probably wrong for github too, so unless there is a good reason to do it like that, let's not